test/TEST-44-LOG-NAMESPACE/testsuite.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -ex
+
+systemd-analyze log-level debug
+
+systemd-run -p LogNamespace=foobar echo "hello world"
+
+journalctl --namespace=foobar --sync
+journalctl --namespace=foobar > /tmp/hello-world
+journalctl > /tmp/no-hello-world
+
+grep "hello world" /tmp/hello-world
+! grep "hello world" /tmp/no-hello-world
+
+systemd-analyze log-level info
```

Comment:
All those new tests added while my test rework is languishing are hurting me. All the rebasing ;(

Reply:
I sympathize!